src/pages/user.tsx

```diff
@@ -26,20 +28,49 @@ const userData = [
     projects: ["Create your Burger", "Learning and Laughing"],
     imageUrl: "https://i.imgur.com/knDwrmG.jpg",
   },
+  {
+    id: 3,
+    name: "Bill Gates",
+    birthDate: "705888000000",
+    position: "Designer UI/UX",
+    salary: 4000,
+    projects: ["My Life Dashboard"],
+    imageUrl: "https://i.imgur.com/v76rx7g.jpg",
+  },
 ];
 
 export function User() {
   const [showModal, setShowModal] = useState(false);
+  const [data, setData] = useState<UserInterface[]>(userData);
```

Comment:
I suggest you put a meaningful name if you're going to have a state to save the data of the fetched users. It could be usersData, or something that makes it easier for people to know.